eng/scripts/Cadl-Project-Generate.ps1

```diff
@@ -42,6 +42,17 @@ $npmWorkingDir = Resolve-Path $tempFolder/$innerFolder
 $mainCadlFile = If (Test-Path "$npmWorkingDir/client.cadl") { Resolve-Path "$npmWorkingDir/client.cadl" } Else { Resolve-Path "$npmWorkingDir/main.cadl"}
 $resolvedProjectDirectory = Resolve-Path "$ProjectDirectory/src"
 
+if (Test-Path "$npmWorkingDir/cadl-project.yaml") {
+    $cadlProjectYaml = Get-Content -Path "$npmWorkingDir/cadl-project.yaml" -Raw
+
+    Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
```

Comment:
This should be removed, we don't want/need to check for the yaml module multiple times in the script.

Reply:
Thanks @benbp I removed it.